README.rst

```diff
@@ -79,6 +93,14 @@ On Ubuntu-based Linux distributions the packages can be installed with given `re
 
 You may need to use *sudo* to allow *apt-get* to install packages dependent upon your access rights.
 
+Install the prerequisites on Red Hat/CentOS
+-------------------------------------------
+
+On Red Hat and CentOS distributions the required packages can be installed automatically with given `requirements.sh <requirements.sh>`_:
+::
+	./requirements.sh
+
+You may need to use *sudo* upon your access rights.
```

Comment:
You should run this as a user with *sudo* access rights.

Reply:
done